lib/docker-shell-command-executor.ts

```diff
@@ -32,7 +33,10 @@ export default class DockerShellCommandExecutor {
   }
 
   async exec(command: string) {
-    await exec(`docker run --rm -v ${this.workingDirectory}:/workdir -w /workdir course-sdk-${this.dockerfileType} ${command}`);
+    let quotedCommand = command.replace(/\\/g, '\\\\')
```

Comment:
What does this do?

Reply:
Globally replaces backslashes, with 2 backslashes, ie their escaped version.

Reply:
Ah, okay makes sense